langchain/src/chains/combine_documents/base.ts

```diff
@@ -21,6 +21,9 @@ export async function formatDocuments({
   documents: Document[];
   config?: RunnableConfig;
 }) {
+  if (documents == null || documents.length == 0) {
```

Comment:
Don't think we need the `.length` check

Also should use triple equals if at all

Reply:
Triple equals added